### ISSUE TITLE
Adding IComparable to cljs-time.extend

### ISSUE THREAD

**Issue by piranha** [andrewmcveigh/cljs-time#42]
I think it would be worth it. I've implemented it in my app like that:

```
(extend-type goog.date.DateTime
  IComparable
  (-compare [o other]
    (- (.getTime o) (.getTime other))))
```

**Comment by pangloss** [comment]
:+1:

**Comment by andrewmcveigh** [comment]
Agree, I'm on holiday at the moment, but back in a week.

**Comment by jwhitbeck** [comment]
+1

**Comment by andrewmcveigh** [comment]
Released as `0.3.13`.
